lib/dagDisplay.mli

```diff
@@ -19,16 +23,17 @@ val make_tree_hts :
   list ->
   (Geneweb_db.Driver.iper, 'a) Def.choice Dag2html.dag ->
   (int * Dag2html.align * Adef.safe_string Dag2html.table_data) array array
-(** [make_tree_hts conf base elem_txt vbar_txt invert set spl d] *)
+(** Build HTML table structure from DAG.
+    [make_tree_hts conf base elem_txt vbar_txt invert set spl dag] *)
 
-val print_slices_menu_or_dag_page :
+val print_dag_page :
   Config.config ->
   Geneweb_db.Driver.base ->
   Adef.safe_string ->
   (int * Dag2html.align * Adef.safe_string Dag2html.table_data) array array ->
   Adef.escaped_string ->
   unit
-(** [print_slices_menu_or_dag_page conf page_title hts next_txt] *)
+(** Render DAG as HTML page using dag.txt template. *)
```

Comment:
The previous documentation gives a name for each argument, which was helpful. Can you explain the meaning of some of them?